integration_tests/cosmoscli.py

```diff
@@ -746,19 +754,21 @@ def gov_propose_legacy(self, proposer, kind, proposal, mode="block", **kwargs):
 
     def gov_vote(self, voter, proposal_id, option, event_query_tx=True, **kwargs):
         kwargs.setdefault("gas_prices", DEFAULT_GAS_PRICE)
-        rsp = json.loads(
-            self.raw(
-                "tx",
-                "gov",
-                "vote",
-                proposal_id,
-                option,
-                "-y",
-                from_=voter,
-                home=self.data_dir,
-                **kwargs,
-            )
+        raw = self.raw(
+            "tx",
+            "gov",
+            "vote",
+            proposal_id,
+            option,
+            "-y",
+            from_=voter,
+            home=self.data_dir,
+            **kwargs,
         )
+        # skip success log info that breaks json.loads
```

Comment:
is it the keyring migration message?

Reply:
yea, separate migrate seems doesn't work